Refresh developer reference (#1219)
prysmaticlabs/documentation: docs/terminology.md

The process of voting on the validity of newly created blocks on the beacon chain.

#### Beacon chain

> The consensus chain that maintains Ethereum's consensus state, coordinates validators, and agrees on the canonical chain. Ethereum does not use the originally proposed execution shard chains; it scales rollups with blob data availability.

Reviewer: ```suggestion
The consensus chain, which maintains Ethereum's consensus state, coordinates validators and agrees on the canonical chain. Ethereum does not use the originally proposed execution shard chains; it scales rollups with blob data availability.
```

A data storage paradigm designed for storing, retrieving, and managing hash tables.

#### Partial validator withdrawal

> An automatic withdrawal of balance above a validator's maximum effective balance to its execution address. The validator remains active. The protocol limits how many withdrawals are processed in each block.

Reviewer: ```suggestion
An automatic withdrawal of the balance above a validator's maximum effective balance to its execution address. The validator remains active. The protocol limits the number of withdrawals processed in each block.
```

#### Proof-of-Stake \(PoS\)

> Ethereum's consensus mechanism, in which validators stake ETH and are selected to propose and attest to blocks. It replaced proof-of-work on Ethereum mainnet.

Reviewer: ```suggestion
Ethereum's consensus mechanism, in which validators stake ETH and are selected to propose and attest to blocks. It replaced proof-of-work on the Ethereum mainnet.
```

The person or entity managing Ethereum validators.

#### Validator

> An on-chain consensus participant identified by a BLS public key. A [validator client](#validator-client) is the software that manages one or more validators.

Reviewer: ```suggestion
An onchain consensus participant identified by a BLS public key. A [validator client](#validator-client) is the software that manages one or more validators.
```
